tools/depends/native/pugixml/Makefile

```diff
@@ -8,15 +8,13 @@ LIBNAME=pugixml
 VERSION=1.10
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.gz
-BASE_URL=https://github.com/zeux/pugixml/releases/download/v$(VERSION)
 
-APP=$(PLATFORM)/pugixml
+APP=$(PLATFORM)/build/libpugixml.a
```

Comment:
To match the other makefiles this should be `LIBNAME` I think, or something with `LIB` instead of `APP`